server/config/development/cdap.json

```diff
@@ -13,5 +13,8 @@
   "dashboard.router.check.timeout.secs": "0",
   "market.base.url": "https://hub.cdap.io/dev/v2",
   "ui.theme.file": "config/themes/default.json",
-  "session.secret.key": "sample-secret-key-for-encryption"
+  "session.secret.key": "sample-secret-key-for-encryption",
+  "feature.lifecycle.management.edit.enabled": "true",
```

Comment:
@sanjanasandeep 6.9 will remove LCM feature flag right? so the UI should add this tag

Reply:
ahh my bad i forgot to delete that

Reply:
yes, this should be enabled

Reply:
No need to remove. Looks like we need it